sdk/nodejs/runtime/settings.ts

```diff
@@ -518,6 +518,15 @@ export async function monitorSupportsDeletedWith(): Promise<boolean> {
     return monitorSupportsFeature("deletedWith");
 }
 
+/**
+ * monitorSupportsAliasSpecs returns a promise that when resolved tells you if the resource monitor we are
+ * connected to is able to support alias specs across its RPC interface. When it does, we marshal aliases
+ * in a special way.
+ */
+export async function monitorSupportsAliasSpecs(): Promise<boolean> {
```

Comment:
Should we mark this as `@internal`? I don't think we'd expect anyone outside this library to use this.

We should probably go back and make the other exported ones `@internal` at some point as well (breaking anyone who happened to use it; so maybe not until a major version bump; though, unlikely anyone else is using them directly).